contracts/MyStrategy.sol

```diff
@@ -347,17 +390,21 @@ contract MyStrategy is BaseStrategy, ReentrancyGuardUpgradeable {
                 // ETH
                 uint256 difference = address(this).balance.sub(beforeBalance[i]);
                 if (difference > 0) {
+                    address recepient = bribesRedirectionPaths[address(WETH)];
```

Comment:
I had originally set it as such but then I realized that because the `_handleRewardTransfer` call at the following [line](https://github.com/Badger-Finance/vested-aura/blob/f8152243d0627e1945b0a6926e2564381caa7faf/contracts/MyStrategy.sol#L398) takes WETH as a parameter (and has to) and because now I am mapping the fee to the token (read from the token passed to this function) then the token mapped to and the token passed to the function had to be the same. Therefore, as long as the fee is mapped to the token (which makes sense), it has to be wETH for ETH. 

Good catch though. I think that as long as it is clear in the docs it should be fine. And again, I don't think we expect to redirect ETH anyways, this is just to have the option to do so.

Reply:
Ah missed that. Makes sense!